src/neural/network_blas.cc

```diff
@@ -220,23 +223,51 @@ class BlasNetwork : public Network {
       Transforms::InvertBatchNormStddev(conv2.bn_stddivs);
     }
 
-    Transforms::OffsetBatchNormMeans(weights_.policy.bn_means, weights_.policy.biases);
+    Transforms::OffsetBatchNormMeans(weights_.policy.bn_means,
+                                     weights_.policy.biases);
     Transforms::InvertBatchNormStddev(weights_.policy.bn_stddivs);
 
-    Transforms::OffsetBatchNormMeans(weights_.value.bn_means, weights_.value.biases);
+    Transforms::OffsetBatchNormMeans(weights_.value.bn_means,
```

Comment:
Is putting them on two lines really necessary? It fits just fine on one line.

Reply:
All the formatting is not mine... (clang-format)

Reply:
That's because https://google.github.io/styleguide/cppguide.html#Line_Length sets line limit of 80 chars.
Very handy, that way I can have three source panes open side by side.